app/service.py

```diff
@@ -33,4 +32,5 @@ def get_customized_message(activity: Activity = None) -> str: # "activity" is an
         "Your mindset determines your outcome; cultivate a winning mentality.",
         "Rise above the noise and negativity; focus on your goals and let nothing stop you."
     ]
+    inp = random.randint(0,int(len(messages)))
```

Comment:
There's a bug in your code :) 

What happens when the inp value will be len(messages)?